.github/workflows/ci.yml

```diff
@@ -513,8 +455,6 @@ jobs:
           flutter drive --target=test_driver/app.dart --dart-define=testName="" --suppress-analytics
         working-directory: ./flutter/realm_flutter/tests
 
-# Android jobs
```

Comment:
these were useful to navigate this big file

Reply:
There's only one android job, so not sure it adds much value anymore.